s3.tf

```diff
@@ -176,7 +176,17 @@ data "aws_iam_policy_document" "bucket" {
 
 }
 
+resource "time_sleep" "wait_before_subscribing" {
```

Comment:
When did this sneak in? Since this is a user-facing change, land it as a separate PR.

Reply:
commented this out, the tests often fail without this because of a race condition

Reply:
reducing a race condition with timeouts isn't optimal, but the cost of trying to remove the race condition doesn't seem worth it